examples/typescript/evm/transfer.ts

```diff
@@ -0,0 +1,35 @@
+// Usage: pnpm tsx evm/transfer.ts
```

Comment:
for consistency, can we call this file `baseAccount.transfer` and update this line accordingly?